src/ml/neural_net/style_transfer/mps_style_transfer_encoding_node.h

```diff
@@ -0,0 +1,24 @@
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h> 
+
+#include <ml/neural_net/style_transfer/mps_style_transfer_utils.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+API_AVAILABLE(macos(10.14))
+@interface TCMPSStyleTransferEncodingNode : NSObject 
+
+@property (nonatomic) MPSNNImageNode *m_output;
+
+- (instancetype) initWithParameters:(NSString *)name
+                          inputNode:(MPSNNImageNode *)inputNode
+                             device:(id<MTLDevice>)dev
+                          cmd_queue:(id<MTLCommandQueue>)cmd_q
```

Comment:
In ObjC code, prefer camelCase. Definitely for parts of selector (method) names, and also for variable/argument names.

In ObjC++ code it can get a bit trickier, for example when a C++ class has a data member whose type is Objective C class. But we don't have to worry about that here.